api/src/lib/utils.js

```diff
@@ -25,7 +25,7 @@ module.exports = class Utils {
     return destination.search('@') > -1
   }
 
-  * webfingerLookup (resource) {
+  * _webfingerLookup (resource) {
```

Comment:
underscore indicates that this is a local function that's not exposed by the "unit", and not tested in the unit tests.